controllers/build/modulehelper.go

```diff
@@ -17,7 +20,7 @@ func NewModuleHelper() ModuleHelper {
 	return moduleHelper{}
 }
 
-func (moduleHelper) ApplyBuildArgOverrides(args []v1alpha1.BuildArg, overrides ...v1alpha1.BuildArg) []v1alpha1.BuildArg {
+func (m moduleHelper) ApplyBuildArgOverrides(args []v1alpha1.BuildArg, overrides ...v1alpha1.BuildArg) []v1alpha1.BuildArg {
```

Comment:
This method doesn't use the `moduleHelper` obj. Should we make it a function instead of a method?

Reply:
I always prefer using methods instead of module functions. Makes code clear and, in addition, makes it easier to write unit tests